src/Skeletor/Api/PackagistApi.php

```diff
@@ -39,11 +40,34 @@ public function getVersionsPackage(string $packageSlug)
     }
 
     /**
+     * @param string $package
+     * @return array
+     */
+    public function searchPackage(string $package)
+    {
+        $data = file_get_contents($this->buildSearchUrl($package));
+        $data = json_decode($data, true);
```

Comment:
should we check $data is valid? something like json_last_error() === JSON_ERROR_NONE ? array_map(...) : []